app/user/rest.py

```diff
@@ -164,8 +164,7 @@ def update_user_attribute(user_id):
             personalisation={
                 'name': user_to_update.name,
                 'servicemanagername': updated_by.name,
-                'email address': user_to_update.email_address,
-                'change_type': update_dct_to_str(update_dct)
```

Comment:
Discovered that the `change_type` is not used in the templates associated with the method.

Can you double-check? Template IDs should be:
```
TEAM_MEMBER_EDIT_EMAIL_TEMPLATE_ID = 'c73f1d71-4049-46d5-a647-d013bdeca3f0'
TEAM_MEMBER_EDIT_MOBILE_TEMPLATE_ID = '8a31520f-4751-4789-8ea1-fe54496725eb'
```

Reply:
Yes!
To find the templates you can go to Service 0, which handles GC Notify's Notifications
https://notification.alpha.canada.ca/services/d6aa2c68-a2d9-4437-ab19-3ae8eb202553/templates/[TEMPLATE_ID]

EDIT_EMAIL has variables ((servicemanagername)) ((name)) and ((email address)), So rather then saying your email was changed it says the new email they changed it too, this is funny because it will go to the new email.

EDIT_MOBILE has only ((servicemanagername)) and of course the number it goes it new phone number. 

So they are clear of the change type and this particular problem.

Reply:
I exactly did that but wanted to make sure I did not miss anything 💯